crates/sui-core/src/unit_tests/move_integration_tests.rs

```diff
@@ -445,6 +447,323 @@ async fn test_object_owning_another_object() {
     assert_eq!(effects.deleted.len(), 2);
 }
 
+#[tokio::test]
+async fn test_entry_point_vector() {
```

Comment:
We also have the `sui-adapter-transactional-tests` 

That might be easier for these, but maybe it's easier to do it programmatically in Rust?
Or is there some other reason you chose for these to be in Rust?

Reply:
Didn't know how to use vectors there (as we discussed elsewhere) but now that I do, I will add at least some tests there.